public/video-ui/src/components/utils/VideoEmbed.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export function VideoEmbed({ sources }) {
+  return (
+    <video className="video-player" controls>
+      {sources.map(source => {
+        return (
+          <source key={source.src} src={source.src} type={source.mimeType} />
```

Comment:
If there is only one source, Safari will barf as it expects a single source to be set as the `src` attribute on the `video`.

We don't target Safari, so happy to keep as is. Maybe add a comment?